core/src/main/scala/kafka/coordinator/GroupMetadata.scala

```diff
@@ -128,6 +129,15 @@ case class GroupSummary(state: String,
                         members: List[MemberSummary])
 
 /**
+ * Case class used to represent group metadata (including the group coordinator) for the DescribeGroup API
+ */
+case class ConsumerGroupSummary(state: String,
+                                protocolType: String,
```

Comment:
I would probably move that check into `describeGroup`. Now that I'm thinking about it, we should probably either rename `describeGroup` to `describeConsumerGroup`, or we should let `describeGroup` return a generic `GroupSummary` while `describeConsumerGroup` returns `ConsumerGroupSummary`.